### PR TITLE
Clean up jsHint warnings

### PR DESCRIPTION
**Pull request by ztoben**
#### Database Migration
NO

#### Description
Clean up jsHint console warnings

#### Todos
No tests needed

#### Issues Fixed or Closed by this PR
None that I'm aware of

### CODE REVIEW

File: src/UI/Settings/NetImport/Options/NetImportOptionsView.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ var view = Marionette.ItemView.extend({
 		var refresh=params.get('refresh');
 		if (oauth && refresh){
 			//var callback_url = window.location.href;
-			history.pushState('object', 'title', (window.location.href).replace(window.location.search, ''));
+			history.pushState('object', 'title', (window.location.href).replace(window.location.search, '')); // jshint ignore:line
```
Comment (ztoben) [inline src/UI/Settings/NetImport/Options/NetImportOptionsView.js]: Only way to clean up this warning is to disable the rule for this line due to history being global in JS and not needing to be defined. I'm not familiar with jsHint so I just disabled linting for the whole line. If someone wants to disable just the `is-not-defined` rule specifically for this line, that would be fine.

**Comment by ztoben** [comment]
Failed on `directory_exist_should_be_able_to_find_existing_folder`. Flaky test? I only touched javascript, so I'm not sure why there would be cs tests failing.

**Comment by galli-leo** [comment]
@ztoben Yeah, those tests randomly fail on windows :/, if all other tests pass it should be good (though the tests don't include the js part).

**Comment by ztoben** [comment]
Okay, I can take a look at that test sometime and see if I can figure it out. My changes should be okay to merge then.

**Pull request merged**